components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,340 @@
+
+## ServiceNow Integration
+
+The integration between a Chef Automate server and a ServiceNow instance requires the following:
+
+- Chef Automate Scoped Application
+- Chef Automate server
+
+The Chef Automate application is a ServiceNow certified scoped application available from the ServiceNow store. The [Chef Automate server](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance, and integrates with the Chef Automate infrastructure and compliance functionality. This server sends HTTPS JSON data feeds to the Chef Automate application in a ServiceNow instance to create and update the _ServiceNow CMDB_ tables.
+
+![Data Flow Diagram](snow_integration_dataflow_diagram.png)
+
+### Prerequisites
+
+- The ServiceNow instance must be publicly reachable on https port 443.
+- [Chef automate server installation](https://docs.chef.io/chef_automate.html).
+- Service now package - System Import Sets com. `glide.system\_import\_set`, min version 1.0.0.
+- Service now package - Configuration Management (CMDB) 1.1.
+- Service now plugin - Configuration Management For Scoped Apps [com.snc.cmdb.scoped] 1.0.0.
+
+## Installation
+### Install Chef Automate application in ServiceNow
+
+The Chef Automate application exposes the REST API endpoint that facilitates the communication between Chef Automate and the ServiceNow instance.
+
+- Visit the ServiceNow store at <https://store.servicenow.com>.
+- Get the Chef Automate application.
+- In the ServiceNow instance, navigate to the **System Applications** > **Applications** menu.
+- From the **Downloads** tab, install the **Chef Automate** application.
+
+### Create Application Users
+
+The application provides several roles appropriate for integration, which can be assigned to existing or new ServiceNow users. The roles are as follows:
+
+- x\_chef\_automate.admin
+- x\_chef\_automate.user
+- x\_chef\_automate.api
+
+These roles are part of the package. Users can create their requisite roles and controls if any further restrictions are required.
+
+#### Role x\_chef\_automate.admin
+
+This role can be assigned to a user other than a Systems administrator to allow administration of the application properties and logs. Thus, administration can be carried out by a user who is not the system administrator. 
+
+Note that a systems administrator can perform all tasks that an `x\_chef\_autoamte.admin` role can. In addition, the Admin role is added by default in ServiceNow and is not part of the app.
+
+The `admin` role grants a user access to the:
+
+- Asset Buffer module
+- Asset Imports module
+- Properties
+- Automate instances.
+- Servers
+- Chef infra client runs
+- Compliance reports profiles
+- Compliance report results
+- Compliance Profiles
+- Compliance Profiles results
+- Asset Queue
```

Comment:
Asset queue should be removed. Its now used anymore

Reply:
Done